### PR TITLE
Extend index.rst page

### LINKED ISSUE #15: Design and add fully functional code example to readme/index.rst
In PR #6 a code example is added. In the future I would like to add an improved example that is fully functional (and tested).

### PR DESCRIPTION
**Pull request by florian-huber**
- Add more extensive example
- Add more explanation to it.
- Add reviewed code example to ``README.rst`` as well.

### CODE REVIEW

File: readthedocs/index.rst [hunk 1]
```diff
@@ -14,29 +14,84 @@ Word2Vec based similarity measure of mass spectrometry data.
...
+Once a word2vec model has been trained, spec2vec allows to calculate the similarities
+between mass spectrums based on this model. In cases where the word2vec model was
+trained on data different than the data it is applied for, a number of peaks ("words")
+might be unknown to the model (if they weren't part of the training dataset). To
+account for those cases it is important to specify the ``allowed_missing_percentage``,
+as in the example below.
+
+.. code-block:: python
+
+    # query_spectrums loaded from files using https://matchms.readthedocs.io/en/latest/api/matchms.importing.load_from_mgf.html
```
Comment (sverhoeven) [inline readthedocs/index.rst:81]: This code block is missing imports. Could you move the unused from code block above to here.

File: readthedocs/index.rst [hunk 2]
```diff
@@ -14,29 +14,84 @@ Word2Vec based similarity measure of mass spectrometry data.
...
 
     # Define similarity_function
-    spec2vec = Spec2VecParallel(model=model, intensity_weighting_power=0.5)
+    spec2vec = Spec2VecParallel(model=model, intensity_weighting_power=0.5,
+                                allowed_missing_percentage=5.0)
 
-    # Calculate scores on all combinations of references and queries
-    scores = list(calculate_scores(references, queries, spec2vec))
+    # Calculate scores on all combinations of reference spectrums and queries
+    scores = list(calculate_scores(spectrums, query_spectrums, spec2vec))
```
Comment (sverhoeven) [inline readthedocs/index.rst]: you import `calculate_scores_parallel` but use `calculate_scores`. Which is it?

**Review by sverhoeven — CHANGES_REQUESTED** [review]

File: readthedocs/index.rst (hunk 1 above)
Comment (florian-huber) [reply readthedocs/index.rst:81]: That's indeed clearer, thanks. 
I changed that and also fixed a error I had in the second code blog.

File: readthedocs/index.rst (hunk 2 above)
Comment (florian-huber) [reply readthedocs/index.rst]: should be ``calculate_scores_parallel``. I just changed it.

**Review by sverhoeven — APPROVED** [review]
Looks OK to me.

Created https://github.com/iomega/spec2vec/issues/21 to test examples

**Comment by florian-huber** [comment]
Nice, thanks Stefan! I will merge it once the tests pass.

**Pull request merged**